### ISSUE TITLE
How to implement Dismissible panel?

### ISSUE THREAD

**Issue by bullet-tooth** [akshathjain/sliding_up_panel#77]
Hey,
Basically I would like to implement a map with the pannel having the following logic:
- Initial state on the panel is hidden
- When the user taps on an object on the map the panel appears in the collapsed(closed) state
- User can close(hide) the pannel by swiping horizontally; or down(if the panel was in the closed state)

I have stuck with implementing described swiping, so I tried:
1. Put `SlidingUpPanel` inside `Dismissable`:
```
    Dismissible(
        child: SlidingUpPanel(
      ...
    ));
```
but,
- Dismissing works only horizontally i.e. setting `sidmissDirection: down` doesn't work
- Dismissing response within a full screen i.e. when swiping on the map, not only on the panel

2. Put `Dismissable` inside `SlidingUpPanel`:
```
    SlidingUpPanel(
      panel: Dismissible(
        child: ...
    ));
```
but, it then an animation becomes broken.

Could you please help with this issue?

**Comment by BenevidesLecontes** [comment]
@bullet-tooth i think you can achieve this, using panel controller and gestureDetector. Panel controller gives you the ability to hide, close and open manually.

**Comment by akshathjain** [comment]
Related to #100.

**Comment by akshathjain** [comment]
Implemented and will be pushed in v1.0.1.

**Comment by Mayb3Nots** [comment]
I dont think this issue has been address. I current cant dismiss the collapsed panel.
